src/data/tagsDataWithDetails.js

```diff
@@ -0,0 +1,178 @@
+// Combines W3 tags and MDN tags, including the deprecated ones
+export const tagsDataWithDetails = `<!--...>, 'Defines a comment', FALSE, '', FALSE,
```

Comment:
Oh wow, I did not expect it to look like this, this is new to me.

I am used to an approach something like this

```
export const tagsDataWithDetails = {
{tag: '<a>', Definition: 'Defines a hyperlink', deprecated: false, equalivent: ', srSupport: true},
{ ... }
```

I cannot even tell all keys for the values 😅 

What are the advantages/disadvantages of each approach?

Reply:
My bad again, I did not explain myself clearly. When I shared the examples with the objects for each tag, that's the way our app "sees" the data (because of the parseTagsData function) but I kept the actual data as a string, just as I imported it from Excel.

Lately I've been working with files in this format, and I admit I didn't stop to think that it's less common. I found it odd in the beginning too, but now I find it's faster to work with. If I need to add some data manually, maybe because of the lack of so many curly braces and no keys, it's much easier to spot what I'm looking for. But again, it is just a matter of being used to it.

We can certainly have the final version of the data as a JSON file. That would be also easier to understand for everyone looking at our files. It will be just a print, copy, paste operation, so not much extra work 😅!

Reply:
No worries. I get it now, true, we are doing the same in Digital Humanities, when importing data from an csv file to create diagrams with it.
In general I do not see a problem using it this way, even so for a beginners project it may be overwhelming. Maybe this way IS common in JavaScript projects as well, but as mentioned, I only know it from Digital Humanities when working with data, like, as a data scientist or so.

I guess, a simple example of an `Object` as a comment would do the thing, where each value has its `key` to understand for people who are not familiar with the project as well. Like I said, taking a first look, I would not know what e.g. the `True`, or `False` or `''` could eventually mean. 

What do you think about it 🤔 ? Or would it then be better do use Objects anyway to avoid these extra explanation. In general, I often read that when you have to explain to much, it means nothing good. But I am not sure in our case, since we use comments on purpose for beginners to understand and learn from it.

Reply:
@YuriDevAT 
I think a JSON would make more sense. It will be easier for everyone looking at the code base. I think we should not discard the string file and the function just yet, but when we wrap up the project, we can decide if they should stay or not.

Reply:
Sounds good.